homeworks/olha.tymchyshyn_olyatmchshn/5-logic/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Basic JS</title>
+        <style>
```

Comment:
If you have a separate file for styles, would be better to keep all styles there.